### ISSUE TITLE
Anchor hover cleared, even `lsp_hover_anchor` is `false `

### ISSUE THREAD

**Issue by dpc** [ul/kak-lsp#128]
I'm using https://gitlab.com/Screwtapello/kakoune-cargo and when I do `cargo-next-error` which displays error in anchored hover, some error messages blink in the hover and quickly disappear. When I disable `kak-lsp` or just not use `lsp-auto-hover-enable` everything is OK. I am guessing that `lsp-auto-hover-enable` is sometimes executed after `kakoune-cargo` and just clears-out the hover.

It's bit odd though because I use `set-option global lsp_hover_anchor false` so all the information that LSP returns is displayed in the right bottom corner, and is not anchored to the cursor. It would be great if I could use both at the same time.

**Comment by Delapouite** [comment]
Probably related to https://github.com/mawww/kakoune/issues/1516

**Comment by dpc** [comment]
Yes. The blinking on the screencap https://github.com/mawww/kakoune/issues/1516#issuecomment-392258943 looks exactly the same.

However, in my case there is often no big hover to display, so maybe if `kak-lsp` would detect "nothing to display" and did not call something, the whole thing would at least happen less often.

**Comment by ul** [comment]
This logic is already there, would you mind posting kak-lsp logs for investigation of failing case?

**Comment by ul** [comment]
No worries, just in case you still get problems on top of mawww/kakoune#1516, I'm keen to investigate, just need a bit of kak-lsp logs for that case.
